Content.Server/Chemistry/EntitySystems/InjectorSystem.cs

```diff
@@ -114,6 +115,12 @@ private void OnInjectorAfterInteract(Entity<InjectorComponent> entity, ref After
     /// </summary>
     private void InjectDoAfter(Entity<InjectorComponent> injector, EntityUid target, EntityUid user)
     {
+        if (HasComp<BlockInjectionComponent>(target)) // DeltaV
+        {
+            Popup.PopupEntity(Loc.GetString("injector-component-deny-user"), target, user);
+            return;
```

Comment:
Additionally, now that chatstacking is a feature we have, we also want to start making things like this also display the message in chat for people who can't read the popup fast enough(Or just miss it..). Generic methods for doing this have been used in several locations in the repo. Similarly, this can use datafields present on the component.

![image](https://github.com/user-attachments/assets/e1f98fee-833f-4b27-b378-31c5c6a773b8)